src/routes/TicketHistory.tsx

```diff
@@ -23,21 +32,82 @@ import printJS from "print-js";
 import { mutation_voidTicket } from "@/lib/graphql/mutations";
 import { useNotificationStore } from "@/lib/stores/notification";
 import { TextArea } from "@/components/TextArea";
+import { Search } from "@/components/Search";
+import { Divider } from "@/components/Divider";
+import clsx from "clsx";
 
 export function TicketHistoryRoute() {
 	useRequireAuth();
 	useDocumentTitle("History");
 	const setNotification = useNotificationStore((s) => s.setNotification);
 
-	//query filters
-	const [fromDate, setFromDate] = useState(new Date().toJSON().slice(0, 10));
-	const [toDate, setToDate] = useState(new Date().toJSON().slice(0, 10));
+	//Setting up default to and from dates
+	const todaysDate = new Date();
+	const daysBack = 30;
+	const defaultFromDate = new Date(todaysDate.getTime() - daysBack * 24 * 60 * 60 * 1000)
+		.toJSON()
+		.slice(0, 10);
+	const defaultToDate = todaysDate.toJSON().slice(0, 10);
+
+	// Date Filters
+	const [fromDate, setFromDate] = useState(defaultFromDate);
+	const [toDate, setToDate] = useState(defaultToDate);
+
+	// get Tickets
+	const ticketsQuery = useTickets({ fromDate: fromDate, toDate: toDate });
+	const tickets = ticketsQuery?.data?.tickets as DeviceTicket[];
+
+	// get UDFs
+	const udfsQuery = useUDFsByType("ticket");
+	const udfs = udfsQuery?.data?.udfs;
 
 	//quick search
-	const [quickSearch, setQuickSearch] = useState<string>();
+	const [quickSearch, setQuickSearch] = useState<string>("");
+
+	// Lane filters
+	const lanesQuery = useLanes();
+	const lanes = lanesQuery?.data?.syncConfigLanes as SyncConfigLane[];
+	const [laneFilter, setLaneFilter] = useState<SyncConfigLane | null>(null);
+
+	// Filtering tickets
+	const [filteredTicketes, setFilteredTickets] = useState<DeviceTicket[]>();
```

Comment:
smol spelling error here. Would be good to keep it the same as your state setter function for clarity.

```suggestion
	const [filteredTickets, setFilteredTickets] = useState<DeviceTicket[]>();
```

Reply:
Ohhh good catch